meteor/override-fastclick.js

```diff
@@ -0,0 +1,6 @@
+// disable fastclick package in Meteor if included
```

Comment:
We'd like to avoid having a `meteor` directory in the root of Ionic. We're not opposed to what it's doing, but is there another way to do it?